src/search/lp/lp_solver.cc

```diff
@@ -37,6 +37,14 @@ using namespace std;
 using utils::ExitCode;
 
 namespace lp {
+#ifndef NDEBUG
+template<class T>
+static bool all_values_are_unique(const vector<T> &v) {
+    set<T> s(v.begin(), v.end());
```

Comment:
```suggestion
    unordered_set<T> s(v.begin(), v.end());
```
since we don't require a sorted container.